_docs/setup/consul/quick-start.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Quick Start on Docker
+overview: Quick Start instructions to setup the Istio service mesh with Docker Compose.
+
+order: 10
+
+layout: docs
+type: markdown
+---
+
+{% include home.html %}
+
+Quick Start instructions to install and configure Istio in a Docker Compose setup.
+
+
+## Prerequisites
+
+* [Docker](https://docs.docker.com/engine/installation/#cloud)
+* [Docker-Compose](https://docs.docker.com/compose/install/)
+
+## Installation steps
+
+1. Go to the [Istio release](https://github.com/istio/istio/releases) page to download the
+   installation file corresponding to your OS. If you are using a MacOS or Linux system, you can also
+   run the following command to download and extract the latest release automatically:
+   ```bash
+   curl -L https://git.io/getLatestIstio | sh -
+   ```
+
+1. Extract the installation file and change the directory to the file location. The
+   installation directory contains:
+    * Sample applications in `samples/`
+    * The `istioctl` client binary in the `bin/` directory. `istioctl` is used for creating routing rules and policies.
+    * The `istio.VERSION` configuration file
+
+1. Add the `istioctl` client to your PATH.
+   For example, run the following command on a MacOS or Linux system:
+
+   ```bash
+   export PATH=$PWD/bin:$PATH
+   ```
+
+1. Generate a configuration file which will be used by `istioctl` and Istio Pilot
+
+    ```bash
+    istioctl context-create --api-server http://172.28.0.13:8080
+    ```
+
+1. Change directory to the root of the Istio installation directory.
+
+1. Bring up the Istio control plane containers:
+
+    ```bash
+    docker-compose -f samples/bookinfo/consul/control-plane.yaml up -d
```

Comment:
Should this be moved to `install/consul/control-plane.yaml`?

Reply:
Yes.. I was thinking the same. Requires a separate PR in istio/istio